Module not found error

The below error is being thrown when the server is started.

Steps to reproduce:
- clone the repo down
- run `asdf install` (to make sure the correct node is installed)
- run `yarn`
- run `yarn dev`

Things tried: 
- updating the `tool-versions` file to `node 18.16.0`
- updating `security-document-loader` to version `0.1.3`
- using `npm i --legacy-peer-deps` to install dependencies instead of yarn


Error:
```
Module not found: Package path . is not exported from package /home/lautom/Desktop/web-verifier-plus/node_modules/@digitalcredentials/crypto-ld (see exports field in /home/lautom/Desktop/web-verifier-plus/node_modules/@digitalcredentials/crypto-ld/package.json)

Import trace for requested module:
./node_modules/@digitalcredentials/security-document-loader/dist/index.js
./node_modules/@digitalcredentials/security-document-loader/dist/esm/index.js
./lib/decode.ts
./pages/index.tsx

https://nextjs.org/docs/messages/module-not-found
```

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@digitalcredentials/crypto-ld": "7.0.3",
+    "@digitalcredentials/crypto-ld": "7.0.6",
     "@digitalcredentials/did-io": "^1.0.2",
     "@digitalcredentials/did-method-key": "^2.0.3",
     "@digitalcredentials/ed25519-signature-2020": "^3.0.1",

